java/src/jmri/jmrix/DCCManufacturerList.java

```diff
@@ -9,52 +48,17 @@
  * @author Bob Jacobsen Copyright (C) 2010
  * @author Kevin Dickerson Copyright (C) 2010
  * @version	$Revision$
- *
+ * @deprecated Since 3.4.5
  */
+@Deprecated
 public class DCCManufacturerList {
 
-    public static final String NONE = "None";  // internal only
-    public static final String LENZ = "Lenz";
-    public static final String HORNBY = "Hornby";
-    public static final String BACHRUS = "Bachrus";
-    public static final String ESU = "ESU";
-    public static final String DCCPP = "DCC++";
-    public static final String DIGITRAX = "Digitrax";
-    public static final String ATLAS = "Atlas";
-    public static final String NCE = "NCE";
-    public static final String CMRI = "C/MRI";
-    public static final String CTI = "CTI Electronics";
-    public static final String EASYDCC = "Easy DCC";
-    public static final String DCC4PC = "DCC4PC";
-    public static final String DCCSPEC = "DCC Specialties";
-    public static final String IEEE802154 = "IEEE 802.15.4";
-    public static final String XBee = "XBee (API Mode)";
-    public static final String JMRI = "JMRI (Network)";
-    public static final String LIONEL = "Lionel TMCC";
-    public static final String MAPLE = "Maple Systems";
-    public static final String MERG = "MERG";
-    public static final String MRC = "MRC";
-    public static final String MARKLIN = "Marklin";
-    public static final String NAC = "NAC Services";
-    public static final String OAK = "Oak Tree Systems";
-    public static final String OPENLCB = "OpenLCB";
-    public static final String OTHER = "Others";
-    public static final String POWERLINE = "Powerline";
-    public static final String PROTRAK = "Protrak";
-    public static final String QSI = "QSI Solutions";
-    public static final String RAIL = "RailDriver";
-    public static final String RFID = "RFID";
-    public static final String PI = "Raspberry Pi Foundation";
-    public static final String ROCO = "Roco";
-    public static final String SPROG = "SPROG DCC";
-    public static final String SRCP = "SRCP";
-    public static final String TAMS = "Tams";
-    public static final String TRACTRONICS = "TracTronics";
-    public static final String UHLEN = "Uhlenbrock";
-    public static final String WANGROW = "Wangrow";
-    public static final String XBEE = "Digi XBee";
-    public static final String ZIMO = "Zimo";
-    public static final String ZTC = "ZTC";
+    //public static final String DCCSPEC = "DCC Specialties"; // phantom? There is no ConnectionTypeList for this name
```

Comment:
This manufacturer used to be listed in the System Connections preferences, but only provided the JMRI Internal connection type. Was it simply miswired within JMRI or should be removed entirely?